org.flatpak.Gourmet.yml

```diff
@@ -0,0 +1,52 @@
+app-id: org.flatpak.Gourmet
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: gourmet
+finish-args:
+  - --share=network
+modules:
+  - name: cpython
+    sources: 
+      - type: archive
+        url:  https://www.python.org/ftp/python/3.8.4/Python-3.8.4.tar.xz
+        sha256:  5f41968a95afe9bc12192d7e6861aab31e80a46c46fa59d3d837def6a4cd4d37
+        
+  - name: intltool
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url:  https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz
+        sha256:  67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd
+
+  - name: dbus-python
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=/app
+    sources:
+      - type: git
+        url:  https://gitlab.freedesktop.org/dbus/dbus-python
+               
+  - name: PrePipDependencies
+    buildsystem: simple
+    build-options:
+      build-args:
+        - --share=network
```

Comment:
I think we would also need ` - --socket=pulseaudio` for the timer alerts

Reply:
You are right, I am trying to figure out where it will work.  I tried it with where pyglet gets installed, but that didn't work just now.

Reply:
got it, it works when added to the run command.  I'll adjust flatpak.yaml to enable the pulseaudio socket.